frontend/src/hooks/useLookupCodes.tsx

```diff
@@ -46,8 +55,34 @@ function useCodeLookups() {
   };
 
   const getOptionsByType = (type: string) => getByType(type).map(mapLookupCode);
+
+  /**
+   * Return an array of SelectOptions containing property classifications.
+   * @function
+   * @param filter - A filter to determine which classifications will be returned.
+   * @returns
+   */
+  const getPropertyClassificationOptions = (
+    filter?: (value: SelectOption, index: number, array: SelectOption[]) => unknown,
+  ) => {
+    const classifications = getByType(API.PROPERTY_CLASSIFICATION_CODE_SET_NAME);
+    return filter
+      ? (classifications ?? []).map(c => mapLookupCode(c)).filter(filter)
+      : !keycloak.hasClaim(Claims.ADMIN_PROPERTIES)
+      ? (classifications ?? [])
```

Comment:
Not a fan of nested ternarys unless they are extremely simple. Suggesting this get simplified.

Reply:
possibly could just remove the filter entirely and allow users to use es6 builtin .map and .filter functions

Reply:
I also agree with nested ternarys being difficult to read. I gave into this one as it was the quickest way to add the overload with the existing code.

Reply:
Removing the filter entirely would remove the ability to override, as you would only get results based on your claims.  The purpose of the override is to allow for this hook to be used in other scenarios.